openstack/cinder/templates/vcenter_datacenter_cinder_configmap.yaml

```diff
@@ -41,6 +41,13 @@ template: |
       volume_backend_name = vmware
       extra_capabilities = '{"vcenter-shard": "{= host.split(".")[0] =}"}'
 
+      [vmware_low_iops]
+      volume_driver = cinder.volume.drivers.vmware.vmdk.VMwareVcVmdkDriver
+      volume_backend_name = vmware_low_iops
+      vmware_storage_profile = cinder-low-iops
```

Comment:
With this, we're in a situation where we can change the name of the `vmware_storage_profile` only for the `vmware` backend, not for the `vmware_low_iops` backend. I'd prefer this to be configurable, too. I'd also prefer if the `vmware_storage_profile` for the `vmware` backend would show up more explicitly. It's currently hidden in the `backend_defaults` section in the for-loop.